knative-serving/config-applications/deployment-resources.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="deployment-resources"]
+= Deployment resources
```

Comment:
Assembly title should match the left-nav title (from `_topic_map.yml`), when possible:
```suggestion
= Configuring deployment resources
```